keyboards/UA62/UA62.c

```diff
@@ -0,0 +1,17 @@
+/* Copyright 2019 NaCly
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "UA62.h"
```

Comment:
Could you please rename all of these files to lowercase? This is to make sure there's no weirdness between OSes that are case sensitive, and ones that aren't.
https://docs.qmk.fm/#/hardware_keyboard_guidelines?id=naming-your-keyboardproject